src/main/flight/servos.c

```diff
@@ -365,11 +366,10 @@ void servoMixer(float dT)
 {
     int16_t input[INPUT_SOURCE_COUNT]; // Range [-500:+500]
 
-    if (FLIGHT_MODE(PASSTHRU_MODE)) {
-        // Direct passthru from RX
-        input[INPUT_STABILIZED_ROLL] = rcCommand[ROLL];
-        input[INPUT_STABILIZED_PITCH] = rcCommand[PITCH];
-        input[INPUT_STABILIZED_YAW] = rcCommand[YAW];
+    if (FLIGHT_MODE(MANUAL_MODE)) {
+        input[INPUT_STABILIZED_ROLL] = rcCommand[ROLL] * currentControlRateProfile->manual_rates[FD_ROLL] / 100L;
```

Comment:
I believe `manual_rates` should be applied when calculating `rcCommand` values in `annexCode`. Mixer is not the right place to apply settings from `controlRateProfile`